Web/CSS2/styleTransform.css

```diff
@@ -0,0 +1,42 @@
+.rectangle{
+	width: 200px;
+	height: 200px;
+    margin:50px;
+	background-color:#5CADFF;
+    border: 3px solid navy;
+    
+}
+
+ 
+.rotate{
+	width: 200px;
+	height: 200px;
+    margin-left:350px;
+	margin-top:-250px;
+	background-color:#5CADFF;
+    border: 3px solid navy;
+    -webkit-transform: rotate(70deg); 
+    transform: rotate(70deg);
+}
+
+.skew{
+	width: 200px;
```

Comment:
Width, height, border and background color are commonly used styles which is defined in rectangle class. Use this class instead of defining these styles in each class.